docusaurus.config.js

```diff
@@ -258,13 +258,6 @@ const config = {
             ],
           },
         ],
-        logo: {
-          alt: 'Deploys by Netlify',
-          src: 'https://netlify.com/img/global/badges/netlify-color-accent.svg',
-          width: 114,
-          height: 51,
-          href: 'https://netlify.com',
-        },
```

Comment:
We can't just remove it, as it's a requirement for the Netlify Open Source plan.  Let's just leave it and wait for it to be fixed.

Reply:
We'll just pay for Netlify. I already put a card behind it back then.